piperider_cli/cloud_connector.py

```diff
@@ -0,0 +1,102 @@
+import os
+import webbrowser
+
+import inquirer
+from rich.console import Console
+from rich.prompt import Prompt
+
+from piperider_cli.cloud import PipeRiderCloud
+from piperider_cli.compare_report import CompareReport
+from piperider_cli.datasource import FANCY_USER_INPUT
+from piperider_cli.filesystem import FileSystem
+
+console = Console()
+piperider_cloud = PipeRiderCloud()
+
+
+def ask_login_info():
+    console.print('Please provide available email account to login')
+    if FANCY_USER_INPUT:
+        account = inquirer.text('Email address', validate=lambda _, x: '@' in x)
+    else:
+        while True:
+            account = Prompt.ask('[[yellow]?[/yellow]] Email address')
+            if '@' in account:
+                break
+
+    response = piperider_cloud.magic_login(account)
+    if response is None or response.get('success') is False:
+        console.print('[[red]Error[/red]] Login failed. Please try again.')
+        return False
+
+    if response.get('link'):
+        webbrowser.open(response.get('link'))
+
+    console.print('Please paste the api token from magic link. The link had be sent to your email address.')
```

Comment:
Suggestion: `The link had be sent to your email address` -> `The link has been sent to your email address`